poster/poster.tex

```diff
@@ -0,0 +1,428 @@
+
+%----------------------------------------------------------------------------------------
+%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
+%----------------------------------------------------------------------------------------
+
+\documentclass[final]{beamer}
+
+\usepackage[scale=1.24]{beamerposter} % Use the beamerposter package for laying out the poster
+
+\usetheme{confposter} % Use the confposter theme supplied with this template
+
+\setbeamercolor{block title}{fg=ngreen,bg=white} % Colors of the block titles
+\setbeamercolor{block body}{fg=black,bg=white} % Colors of the body of blocks
+\setbeamercolor{block alerted title}{fg=white,bg=dblue!70} % Colors of the highlighted block titles
+\setbeamercolor{block alerted body}{fg=black,bg=dblue!10} % Colors of the body of highlighted blocks
+% Many more colors are available for use in beamerthemeconfposter.sty
+
+%-----------------------------------------------------------
+% Define the column widths and overall poster size
+% To set effective sepwid, onecolwid and twocolwid values, first choose how many columns you want and how much separation you want between columns
+% In this template, the separation width chosen is 0.024 of the paper width and a 4-column layout
+% onecolwid should therefore be (1-(# of columns+1)*sepwid)/# of columns e.g. (1-(4+1)*0.024)/4 = 0.22
+% Set twocolwid to be (2*onecolwid)+sepwid = 0.464
+% Set threecolwid to be (3*onecolwid)+2*sepwid = 0.708
+
+\newlength{\sepwid}
+\newlength{\onecolwid}
+\newlength{\twocolwid}
+\newlength{\threecolwid}
+\setlength{\paperwidth}{48in} % A0 width: 46.8in
+\setlength{\paperheight}{36in} % A0 height: 33.1in
+\setlength{\sepwid}{0.024\paperwidth} % Separation width (white space) between columns
+\setlength{\onecolwid}{0.22\paperwidth} % Width of one column
+\setlength{\twocolwid}{0.464\paperwidth} % Width of two columns
+\setlength{\threecolwid}{0.708\paperwidth} % Width of three columns
+\setlength{\topmargin}{-0.5in} % Reduce the top margin size
+%-----------------------------------------------------------
+
+\usepackage{graphicx}  % Required for including images
+\newcommand{\Cyclus}{\textsc{Cyclus}\xspace}%
+
+\usepackage{tabularx}
+\newcolumntype{b}{X}
+\newcolumntype{s}{>{\hsize=.5\hsize}X}
+\newcolumntype{m}{>{\hsize=.75\hsize}X}
+\newcolumntype{z}{>{\hsize=.65\hsize}X}
+
+\usepackage{booktabs} % Top and bottom rules for tables
+
+
+\usepackage{tikz}
+\usetikzlibrary{positioning, arrows, decorations, shapes }
+% Define block styles
+\tikzstyle{decision} = [diamond, draw, fill=blue!20, 
+text width=4.5em, text badly centered, node distance=3cm, inner sep=0pt]
+
+
+\tikzstyle{block} = [rectangle, draw, text centered, fill=blue!20]
+\tikzstyle{line} = [draw, -latex']
+\tikzstyle{cloud} = [draw, ellipse,fill=red!20, node distance=6em,
+minimum height=2em]
+
+
+
+\usetikzlibrary{shapes.multipart}
+\usetikzlibrary{positioning}
+
+
+\setbeamertemplate{bibliography item}[text]
+
+\bibliographystyle{abbrv}
+\bibliography{poster.bib}
```

Comment:
please just use "poster" rather than poster.bib. Also, poster.bib is completely missing from this repository, so please git add it : 
```
I found no \bibdata command---while reading file poster.aux
Warning--I didn't find a database entry for "boucher_international_2010"
Warning--I didn't find a database entry for "brown_identification_2016"
Warning--I didn't find a database entry for "mccarthy_benchmark_2012"
Warning--I didn't find a database entry for "carre_overview_2009"
Warning--I didn't find a database entry for "hoffman_expanded_2016"
Warning--I didn't find a database entry for "yanex_agent-based_2009"
Warning--I didn't find a database entry for "gonzalez_detailed_2002"
Warning--I didn't find a database entry for "kusiak_data-driven_2010"
Warning--I didn't find a database entry for "shebalov_practical_2009"
Warning--I didn't find a database entry for "guerin_impact_2009"
Warning--I didn't find a database entry for "mouginot_class_2012"
Warning--I didn't find a database entry for "coquelet-pascal_cosi6:_2015"
Warning--I didn't find a database entry for "huff_fundamental_2016"
Warning--I didn't find a database entry for "van_den_durpel_daness:_2006"
Warning--I didn't find a database entry for "park_modeling_2003"
Warning--I didn't find a database entry for "feng_standardized_2016"
Warning--I didn't find a database entry for "schneider_nfcsim:_2005"
Warning--I didn't find a database entry for "aubin_development_2013"
Warning--I didn't find a database entry for "garrick_nuclear_2011"
Warning--I didn't find a database entry for "iaea_nuclear_2007"
Warning--I didn't find a database entry for "woodard_stationarity_2011"
Warning--I didn't find a database entry for "li_kernal_2016"
Warning--I didn't find a database entry for "edmonds_advanced_1994"
Warning--I didn't find a database entry for "fishbone_markal_1981"
Warning--I didn't find a database entry for "ansari_predicting_2015"
(There was 1 error message)
```